src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/ClaimantCCJRequestService.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.documents.CCJByAdmissionOrDeterminationPdfService;
+import uk.gov.hmcts.cmc.claimstore.documents.output.PDF;
+import uk.gov.hmcts.cmc.claimstore.events.ccj.CountyCourtJudgmentEvent;
+import uk.gov.hmcts.cmc.claimstore.services.document.DocumentsService;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import static uk.gov.hmcts.cmc.domain.models.CountyCourtJudgmentType.ADMISSIONS;
+import static uk.gov.hmcts.cmc.domain.models.CountyCourtJudgmentType.DETERMINATION;
+
+@Service
+@ConditionalOnProperty("feature_toggles.ctsc_enabled")
+public class ClaimantCCJRequestService {
+
+    private final CCJByAdmissionOrDeterminationPdfService ccjByAdmissionOrDeterminationPdfService;
+    private final DocumentsService documentService;
+
+    @Autowired
+    public ClaimantCCJRequestService(
+        CCJByAdmissionOrDeterminationPdfService ccjByAdmissionOrDeterminationPdfService,
+        DocumentsService documentService
+    ) {
+        this.ccjByAdmissionOrDeterminationPdfService = ccjByAdmissionOrDeterminationPdfService;
+        this.documentService = documentService;
+    }
+
+    @EventListener
+    public void getAndSaveDocumentToCcd(CountyCourtJudgmentEvent event) {
```

Comment:
Please rename this method. I don't see getting document from anywhere in this implementation.  We are creating a PDF document and then uploading to document store(it's not CCD).